models/user.js

```diff
@@ -6,18 +6,23 @@ const { Schema } = mongoose;
 const userModel = new Schema({
   method: {
     type: String,
-    enum: ['local'],
+    enum: ['local', 'facebook'],
     required: true
   },
   local: {
     email: {
-      type: String,
-      required: true
+      type: String
     },
     password: {
-      type: String,
-      required: true
+      type: String
     }
+  },
+  facebook: {
+    id: String,
+    email: String,
+    full_name: String,
```

Comment:
do we need these names fields in the database? if yes then please change the case to camel case of these variables.